src/planscape/impacts/services.py

```diff
@@ -81,3 +82,50 @@ def upsert(treatment_plan, project_area, stand, action_type, user):
         )
     )
     return results
+
+
+@transaction.atomic()
+def clone_treatment_prescription(
+    rx_prescription: TreatmentPrescriptionEntityType,
```

Comment:
Is this supposed to be tx_ or rx_ here? (I think I was using rx as shorthand for prescription in an internal test, but wasnt sure if this was the intent here.)